Problem5/test/problem5/Problem5Test.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem5;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author manxboy
+ */
+public class Problem5Test {
+    @Test
+    public void testSearch() {
```

Comment:
Should have additional tests that don't use the provided word list.  It would allow you cover edge cases like:
- empty array
- single word array
- word list where there are no letter starting with a particular character (i.e. no words that start with 'c')